app/admin/jobs_admin.rb

```diff
@@ -115,13 +68,15 @@
     def scoped_collection
       if params['code_location_id']
         CodeLocation.find(params['code_location_id']).jobs
-      elsif params[:project_id]
-        project_jobs
       else
         super
       end
     end
 
+    def index
+      redirect_to oh_admin_project_jobs_path if params[:project_id]
+    end
+
```

Comment:
This was kept for the convenience thing :) ..The admins would have the history of project jobs and thought it would be handy and also if they land from other links (if any) would still be redirected. I am OK if its needs to be removed